core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

```diff
@@ -167,3 +173,55 @@ fun LpOutlinedTextFieldNumber(
         keyboardOptions = KeyboardOptions.Default.copy(keyboardType = KeyboardType.Number)
     )
 }
+
+/**
+ * This function creates a phone number OutlinedTextField
+ * @param modifier Set component modifier
+ * @param showError This parameter determines whether the error is displayed or not
+ * @param onPhoneChange Returns value typed
+ *
+ * @author Yaritza Moreno
+ */
+@ExperimentalMaterial3Api
+@Composable
+fun PhoneNumberText(
+    modifier: Modifier = Modifier,
+    value: String,
```

Comment:
Please be more descriptible of what is for the variables in the name for these: value, text.